docs/content/preview/benchmark/github.md

```diff
@@ -0,0 +1,22 @@
+<!--
++++
+private=true
++++
+-->
+
+Install sysbench using the following steps:
+
+```sh
+$ cd $HOME
+$ git clone https://github.com/yugabyte/sysbench.git
+$ cd sysbench
+$ ./autogen.sh && ./configure --with-pgsql && make -j && sudo make install
+```
+
+This installs the sysbench utility in `/usr/local/bin`.
+
+Make sure you have the [YSQL shell](../../api/ysqlsh/) `ysqlsh` exported to the `PATH` variable.
```

Comment:
Does this not apply to other OSs?

Reply:
@shaharuk-yb, can you provide an answer to this?

Reply:
@sudhanshu456 I think this is resolved. I don't see it as part of diff anymore.